libnvidia-container.yaml

```diff
@@ -0,0 +1,83 @@
+package:
+  name: libnvidia-container
+  version: 1.15.0
+  epoch: 0
+  description: NVIDIA container runtime library
+  copyright:
+    - license: Apache-2.0
+    - license: GPL-3.0-only
+    - license: LGPL-3.0-or-later
+
+environment:
+  contents:
+    packages:
+      - automake
+      - bmake
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - coreutils # for GNU date
+      - curl
+      - glibc-dev
+      - go
+      - libcap-dev # for sys/capability.h
+      - libseccomp
+      - libseccomp-dev
+      - libtirpc-dev # for rpc/rpc.h
+      - libtool
+      - linux-headers
+      - lsb-release-minimal
+      - m4
+      - nfs-utils # for rpcgen
+      - patch
+      - pkgconf-dev
+  environment:
+    CPATH: /usr/include/tirpc
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/NVIDIA/libnvidia-container
+      tag: v${{package.version}}
+      expected-commit: 6c8f1df7fd32cea3280cf2a2c6e931c9b3132465
+
+  # Fix prefix to /usr instead of /usr/local
+  - runs: sed -i 's|export prefix *= */usr/local|export prefix = /usr|' Makefile
+
+  - uses: autoconf/make
+    with:
+      opts: |
+        WITH_TIRPC=yes \
+
+  - uses: autoconf/make-install
+
+  - uses: strip
+
+update:
+  enabled: true
+  ignore-regex-patterns:
+    - rc*
+  github:
+    identifier: NVIDIA/libnvidia-container
+    strip-prefix: v
+
+test:
+  environment:
+    contents:
+      packages:
+        - posix-libc-utils
+        - sudo-rs
+        - shadow # for useradd
+  pipeline:
+    - name: Ensure the libraries are linked correctly
+      runs: |
+        ldd /usr/lib/libnvidia-container.so
+        ldd /usr/lib/libnvidia-container-go.so
+    - name: Test nvidia-container-cli
+      runs: nvidia-container-cli --version
+    - name: Test nvidia-container-cli with privileged mode
+      runs: |
+        set +e
+        useradd nvidia
+        # This is the expected error since `libnvidia-ml.so` must be available on the host system. Library can be found in the CUDA.
```

Comment:
Could the package that provides this be added? I think there's value in an affirmative test that can prove this package is able to load its dependencies.

Reply:
We're going to handle this case in the image. 👍